### PR TITLE
remove dynamic truncation from tables

### PR DESCRIPTION
**Pull request by isstuev**
resolves https://github.com/blockscout/frontend/issues/1575

### CODE REVIEW

File: ui/pages/__screenshots__/NameDomain.pw.tsx_default_history-tab-mobile-1.png
Comment (tom2drum) [inline ui/pages/__screenshots__/NameDomain.pw.tsx_default_history-tab-mobile-1.png:1]: Is the absence of the "tx" icon intentional?

File: ui/pages/__screenshots__/Withdrawals.pw.tsx_default_base-view-mobile-1.png
Comment (tom2drum) [inline ui/pages/__screenshots__/Withdrawals.pw.tsx_default_base-view-mobile-1.png:1]: The same question about block's icon here.

File: ui/tx/details/__screenshots__/TxInfo.pw.tsx_default_with-token-transfer-mobile-1.png
Comment (tom2drum) [inline ui/tx/details/__screenshots__/TxInfo.pw.tsx_default_with-token-transfer-mobile-1.png:1]: I think the previous screenshot was better when every minted token was on one line.

**Review by tom2drum — APPROVED** [review]

File: configs/envs/.env.optimism_goerli [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+NEXT_PUBLIC_NETWORK_EXPLORERS=[{'title':'Etherscan','baseUrl':'https://etherscan.io/','paths':{'tx':'/tx','address':'/address','token':'/token','block':'/block'}}]
+# app features
+NEXT_PUBLIC_APP_INSTANCE=local
+NEXT_PUBLIC_APP_ENV=development
+NEXT_PUBLIC_GRAPHIQL_TRANSACTION=0xf7d4972356e6ae44ae948d0cf19ef2beaf0e574c180997e969a2837da15e349d
+NEXT_PUBLIC_IS_ACCOUNT_SUPPORTED=true
+# NEXT_PUBLIC_AUTH_URL=http://localhost:3000
+NEXT_PUBLIC_API_WEBSOCKET_PROTOCOL=ws
+NEXT_PUBLIC_LOGOUT_URL=https://blockscoutcom.us.auth0.com/v2/logout
+NEXT_PUBLIC_STATS_API_HOST=https://stats-eth-main.k8s.blockscout.com
```
[the remark this answers is not in the document]
Comment (isstuev) [reply configs/envs/.env.optimism_goerli:41]: idk, just copy pasted it from the eth config. what would be the right one?

File: ui/pages/__screenshots__/NameDomain.pw.tsx_default_history-tab-mobile-1.png
Comment (isstuev) [reply ui/pages/__screenshots__/NameDomain.pw.tsx_default_history-tab-mobile-1.png:1]: yes, we don't have icons (except addresses) in our common tables (like transactions and blocks), so I tried to unify the views for all the other tables

File: ui/tx/details/__screenshots__/TxInfo.pw.tsx_default_with-token-transfer-mobile-1.png
Comment (isstuev) [reply ui/tx/details/__screenshots__/TxInfo.pw.tsx_default_with-token-transfer-mobile-1.png:1]: fixed a bit, but I don't see a point in keeping it one lone for 100% cases, I think it's better to break line if there's not enough space, that have only one letter in token name

File: configs/envs/.env.optimism_goerli (hunk 1 above)
[the remark this answers is not in the document]
Comment (tom2drum) [reply configs/envs/.env.optimism_goerli:41]: The `https://stats-optimism-goerli.k8s.blockscout.com` is used on the prod instance. So it should be the same here, I guess.

**Pull request merged**